charts/model-engine/templates/service_template_config_map.yaml

```diff
@@ -285,6 +285,12 @@ data:
                   port: ${USER_CONTAINER_PORT}
                 initialDelaySeconds: ${READINESS_INITIAL_DELAY}
                 periodSeconds: 5
+              livenessProbe:
+                httpGet:
+                  path: ${HEALTHCHECK_ROUTE}
+                  port: ${USER_CONTAINER_PORT}
+                initialDelaySeconds: ${READINESS_INITIAL_DELAY}
+                timeoutSeconds: 5
```

Comment:
Might be worth putting a comment saying that 5s helps for vLLM under load, per your PR description. Maybe same for the `readinessProbe`?